wimprates/halo.py

```diff
@@ -7,10 +7,20 @@
 from scipy.special import erf
 
 
+
 import wimprates as wr
 export, __all__ = wr.exporter()
 
 
+# See https://arxiv.org/abs/2105.00599 and references therein
+_HALO_DEFAULTS = dict(
+    rho_dm = 0.3, # GeV / c2 / cm3
+    v_esc = 544,  # km/s
+    v_orbit = 29.8,  # km/s
+    v_pec = (11.1, 12.2, 7.3),  # km/s
+    v_0 = 238,  # km/s
+)
```

Comment:
probably easier this way to bookkeep and prevent other occurances

Reply:
Sure. The variables don't seem to be repeated but maybe they will be in the future. People could try to modify the dict at runtime, and we haven't tested whether that gives consistent behavior -- but monkeying around with things in underscored variables voids warranty anyway.